src/lunamary/services/CommonService.java

```diff
@@ -0,0 +1,90 @@
+package lunamary.services;
+
+import datastructures.circulardoublylinkedlist.MyCircularDoublyLinkedList;
+import lunamary.modelPerson.Parent;
+import lunamary.modelPerson.Student;
+import lunamary.modelPerson.Teacher;
+import lunamary.modelSchool.Classroom;
+import lunamary.modelSchool.School;
+import lunamary.modelSchool.Subject;
+
+public class CommonService {
+
+
+    public CommonService() {
+
+    }
+
+    public void print(String text) {
+        System.out.print(text);
+    }
+
+
+    public static Classroom getClassroom(School school, String codeClassroom) {
```

Comment:
If you implement singleton correctly there is no need to pass school entities all over the place.